### PR TITLE
Parks and gardens

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Linked to QwantResearch/openmaptiles#19

### CODE REVIEW

File: kartotherian/Dockerfile [hunk 1]
```diff
@@ -16,7 +16,7 @@ RUN git clone https://github.com/QwantResearch/kartotherian.git /opt/kartotheria
     && npm install --production \
     && git clone https://github.com/QwantResearch/kartotherian_config.git /opt/kartotherian_config \
     && cd /opt/kartotherian_config \
-    && git checkout 78208ef48b925c1e81fdad4dd47227b3d31f7fe1
+    && git checkout 6bc774c19bd236ca6a362478bfcb556a69455d97
```
Comment (amatissart) [inline kartotherian/Dockerfile]: please update this commit to use the latest master now that https://github.com/QwantResearch/kartotherian_config/pull/48 is merged

File: README.md [hunk 2]
```diff
@@ -5,6 +5,20 @@ docker images for the [kartotherian project](https://github.com/kartotherian/kar
 They use a mixed architecture of Kartotherian and [openmaptiles](https://github.com/openmaptiles/openmaptiles)
 You need a docker version > 18.06
 
+## Workflow
```
Comment (amatissart) [inline README.md]: thanks :+1: 

maybe we could move this section after the "quick start" instructions ?

Comment (GuillaumeGomez) [reply README.md]: I find it easier to miss if put after. In this case, we just give a quick overlook and that's it.

**Comment by GuillaumeGomez** [comment]
Updated.

**Pull request merged**